webgl/lessons/ko/webgl-fundamentals.md

```diff
@@ -466,7 +472,7 @@ clip 공간에서 왼쪽 하단 모서리는 -1, -1 입니다.
 우린 아직 어떤 부호도 바꾸지 않았기 때문에 현재 0, 0은 좌측 하단이 됩니다.
 2D 그래픽 API를 사용해서 전통적인 왼쪽 상단 모서리를 얻으려면 clip 공간 y좌표를 뒤집어서 사용할 수 있습니다.
 
-    *   gl_Position = vec4(clipSpace * vec2(1, -1), 0, 1);
+    *gl_Position = vec4(clipSpace * vec2(1, -1), 0, 1);
```

Comment:
`*` at the beginning of a line of code tells the prettyify function to mark the line as changed. There's 3 codes. 

`-` means line deleted
`*` means line changed
`+` means line added

Then, because of markdown, those either need to be the 5th character in code block (since the preceding 4 spaces designate the block) or the first character if using triple backtick style

Reply:
Yes so there are 4 spaces front of ```*```.
And look like this:
![2018-11-07 2 28 28](https://user-images.githubusercontent.com/20078201/48112507-0b772a00-e29a-11e8-9e1b-c3606a4bbe35.png)
Maybe I'm not understand your think.
Can you explain it more detail?

Reply:
Oops, I guess I saw it wrong